mk/package-and-sign.sh

```diff
@@ -53,13 +52,7 @@ mkdir_clean()
 {
   rm -rf $1 && mkdir -p $1
 }
-
-#overwrite sign file
```

Comment:
We still need to overwrite the sign file, because someone might want to use their own sign file, not the one packed by us in XenAdminUnsigned.zip